entities/config.go

```diff
@@ -1,10 +1,12 @@
 package entities
 
-// Typedef for the yaml config object.
-type Config_t struct {
-	MotherShipIP   string `toml:"motherShipIP"`
-	MotherShipPort string `toml:"motherShipPort"`
-	MotherShipSSL  bool   `toml:"motherShipSSL"`
+// Typedef for the toml config objects.
+type ClientConfig_t struct {
+	IP    string `toml:"ip"`
```

Comment:
Wo ist mothership?????

Reply:
It got crazy to write conditions on these laaarge keys 😩